bit9PlatformAPI/examples/samples/TranslateSyncFlags.py

```diff
@@ -0,0 +1,30 @@
+import sys
+import os
+import time
+
+# Includes the "common" folder that comes from GitHub
+sys.path.append(os.path.join(os.path.dirname(os.path.dirname(__file__)), 'common'))
+import bit9api
+
+bit9 = bit9api.bit9Api(
+    "https://bit9.server.xyz",  # Replace with actual Bit9 server URL
+    token="api_token",  # Replace with actual Bit9 user token for VT integration
+    ssl_verify=False  # Don't validate server's SSL certificate. Set to True unless using self-signed cert on IIS
+)
+
+# Find all computers with the specified name that are connected
+comps = bit9.search('v1/computer', ['name:DOMAIN\NAME', 'connected:true'])
```

Comment:
Can you generalize this into two parts:

1. A function that can be reused by others (eg. `def convert_sync_flags(sync_flags)` which takes the hex value from the computer's `syncFlags` attribute and returns a list of strings with the converted flag values)
2. Refactor into a simple script driver that takes a command line parameter describing the search parameters (computer name, etc.) that are passed to the `bit9.search()` function. Then that main function performs the search and passes the syncFlags into the function defined above, to show the proper usage of the function, and formats the output as you have here.

Thanks!